packages/utilities/src/modalize.ts

```diff
@@ -11,27 +11,30 @@ import { getDocument } from './dom/getDocument';
  * Returns a function that undoes the changes it made.
  */
 export function modalize(target: HTMLElement): () => void {
-  let affectedNodes: HTMLElement[] = [];
-  const targetDocument = getDocument(target) || document;
+  const targetDocument = getDocument(target);
+  if (!targetDocument) {
+    // can't do this in SSR
+    return () => undefined;
+  }
+
+  let affectedNodes: [HTMLElement, string | null][] = [];
 
   // start at target, then recurse and do the same for parent, until we reach <body>
-  while (target !== targetDocument.body) {
+  while (target !== targetDocument.body && target.parentElement) {
```

Comment:
is this to check that `target` isn't the document node? I'm trying to think of another case where `el.parentElement` would be falsy

Reply:
oh wait, just to avoid the `!` later? nvm :D